src/hotspot/share/classfile/placeholders.cpp

```diff
@@ -43,21 +43,21 @@
 // result the first thread gets.
 class SeenThread: public CHeapObj<mtInternal> {
 private:
-   Thread *_thread;
+   JavaThread* _thread;
    SeenThread* _stnext;
    SeenThread* _stprev;
 public:
-   SeenThread(Thread *thread) {
+   SeenThread(JavaThread* thread) {
        _thread = thread;
        _stnext = NULL;
        _stprev = NULL;
    }
-   Thread* thread()                const { return _thread;}
-   void set_thread(Thread *thread) { _thread = thread; }
+   JavaThread* thread()                const { return _thread;}
+   void set_thread(JavaThread* thread) { _thread = thread; }
```

Comment:
Can you line these { up also?

Reply:
Sure. The above matched the style on that little block of code but I can change both bits.

Thanks for the review.